packages/lib-classifier/src/store/feedback/helpers/metadata-to-rules.spec.js

```diff
@@ -5,10 +5,10 @@ describe('feedback: metadataToRules', function () {
   function mockSubjectWithRule (ruleID) {
     return {
       metadata: {
-        '#feedback_1_id': ruleID,
-        '#feedback_1_answer': '0',
-        '#feedback_1_failureMessage': 'Actually, this sound is from noise (background)',
-        '#feedback_1_successMessage': 'Correct!'
+        '#feedback_[1]_id': ruleID,
```

Comment:
I should prob add a comment or warning to explicitly suggest NOT wrapping the `ruleIndex` in brackets. Maybe there should be two sets of similar tests, one set for `ruleIndex` wrapped in brackets one set for it not?

Reply:
The instructions for feedback explicitly state that Nin `feedback_N` should be an integer. https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/features/feedback/README.md

Reply:
Having a spec for the case where N isn’t an integer would be good. Maybe update the spec to test subjects that conform to the rules and also subjects that don’t conform.